devtools_m2/cypress/support/commands.js

```diff
@@ -28,7 +28,7 @@ afterEach(function() {
   // https://github.com/cypress-io/cypress/issues/216
   // https://github.com/cypress-io/cypress/issues/686
   cy.log('Reset web interface to avoid bleed-over')
-  cy.visit('/lib/web/blank.html', { failOnStatusCode: false })
+  cy.visit('/lib/web/blank.html', { failOnStatusCode: false  })
```

Comment:
Nit: Could you fix this whitespace?

Reply:
please tell me you have some sort of automation around catching this...

Reply:
Not yet. But we could pretty easily throw eslint around these files.